source/common/ssl/utility.cc

```diff
@@ -18,5 +21,31 @@ std::string Utility::getSerialNumberFromCertificate(X509& cert) {
   return "";
 }
 
+const std::vector<std::string> Utility::getSubjectAltNames(X509& cert) {
```

Comment:
I don't think this function is needed. There is already getDnsSansFromCertificate() and getUriSanFromCertificate(). You can just combine the results from those two.

Reply:
Are you referring to these functions?
https://github.com/envoyproxy/envoy/blob/f936fc60f488cfae07f5e5d20d7381f0f23482fe/source/common/ssl/ssl_socket.h#L72
There is a TODO to move them to Utility. So I have added them in Utitlity now and will do a follow-up PR to use these functions there as well. WDYT? If you are referring to some thing else, please let me know. I will use them.